### PR TITLE
docs: Update user guide

### PR DESCRIPTION
**Pull request by m1oojv**
## What type of PR is this? (check all applicable)

-   [ ] Refactor
-   [ ] Feature
-   [ ] Bug Fix
-   [ ] Optimization
-   [x] Documentation Update

## Description

-  Update user guide with deleteAllTask, markTask and unmarkTask

## Related Tickets & Documents

-   Related Issue #38 

## Added/updated tests?

_We encourage you to keep the code coverage percentage at 75% and above for new changes._

-   [ ] Yes
-   [x] No, and this is why: 
-   [ ] No, there are UI changes that I'm unable to cover.
-   [ ] I need help with writing tests

### CODE REVIEW

File: docs/DraftUserGuide.md [hunk 1]
```diff
@@ -417,8 +417,93 @@ Errors:
...
+```
+markTask INDEX
+```
+
+- Marks the task at the specified `INDEX`.
+- The index refers to the index number shown in the displayed task list.
+- The index **_must be a positive integer_** 1, 2, 3, …
+
+{% include admonition.html type="info" title="Tasks start off as works in progress incomplete by default." %}
+
```
Comment (jiakai-17) [inline docs/DraftUserGuide.md:458]: Perhaps you could improve on the phrasing of this part?

```suggestion
{% include admonition.html type="info" title="

Tasks are marked as incomplete by default.

" %}
```

File: docs/DraftUserGuide.md [hunk 2]
```diff
@@ -417,8 +417,93 @@ Errors:
...
+Output:
+
+![deleteAllTask success](images/output/deleteAllTask_success.png)
+
+<div style="page-break-after: always;"></div>
+
+### 14. Marking a task as completed: `markTask`
+
+Allows you to indicate that a specific task **has been finished**.
```
Comment (jiakai-17) [inline docs/DraftUserGuide.md:445]: Perhaps you could replace `finished` with `completed` to standardise naming?

```suggestion
Allows you to indicate that a specific task **has been completed**.
```

File: docs/DraftUserGuide.md [hunk 3]
```diff
@@ -417,8 +417,93 @@ Errors:
...
+
+```
+unmarkTask INDEX
+```
+
+- Marks the task at the specified `INDEX`.
+- The index refers to the index number shown in the displayed task list.
+- The index **_must be a positive integer_** 1, 2, 3, …
+
+{% include admonition.html type="info" title="Tasks start off as works in progress incomplete by default." %}
```
Comment (jiakai-17) [inline docs/DraftUserGuide.md:488]:
```suggestion
{% include admonition.html type="info" title="

Tasks are marked as incomplete by default.

" %}
```

File: docs/DraftUserGuide.md [hunk 4]
```diff
@@ -417,8 +417,93 @@ Errors:
...
+
+Errors:
+
+![markTask_error](images/error/markTask_error.png)
+
+<div style="page-break-after: always;"></div>
+
+### 14. Marking a task as incomplete: `unmarkTask`
+
+Allows you to indicate that a specific task **has not been finished**.
```
Comment (jiakai-17) [inline docs/DraftUserGuide.md:476]:
```suggestion
Allows you to indicate that a specific task **has not been completed**.
```

**Review by jiakai-17 — CHANGES_REQUESTED** [review]
Thanks for the changes, just some minor nits about phrasing.

**Review by jiakai-17 — APPROVED** [review]

**Review by freshcabbage123 — APPROVED** [review]
LGTM, thank you for amending the phrasing to make it more user-centric!

**Pull request merged**